server/routes/bidding.py

```diff
@@ -0,0 +1,34 @@
+from flask import Blueprint, g, request
+from datetime import date, timedelta
+from src.bidding import BIDDERS
+
+bp = Blueprint("bid", __name__, url_prefix="/bid")
+
+
+@bp.route("/list", methods=["GET"])
+def list_bidders():
+    try:
+        keys = list(BIDDERS.keys())
+        if "default" in keys:
+            keys.remove("default")
+        default = "none"
+        for key in keys:
+            if BIDDERS[key] == BIDDERS.get("default"):
+                default = key
+        mssg = {"bidders": keys, "default": default}
+        return mssg, 200
+    except Exception as e:
+        return {"message": str(e)}, 500
```

Comment:
Ah so the endpoint lets you choose which bidding algorithm to use?

Reply:
that's my plan, but we can definitely change it in the future